airflow/models.py

```diff
@@ -2609,7 +2611,10 @@ def get_extra_links(self, dttm, link_name):
             one of the options specified in `extra_links`
         :return: A URL
         """
-        pass
+
+        if link_name in type(self).extra_link_functions:
+            call_func = type(self).extra_link_functions[link_name]
+            return call_func(self, dttm)
```

Comment:
If I overwrite `get_extra_links`, this stops working. I think if we're going to do a function lookup table, we just do the lookup table.

Reply:
I agree.